pkg/cmd/generate-test-suites/main.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"sort"
+	"strings"
+)
+
+func main() {
+	// First list all tests.
+	buf, err := exec.Command("bazel", "query", "kind(go_test, //pkg/...)", "--output=label").Output()
+	if err != nil {
+		log.Printf("Could not query Bazel tests: got error %v", err)
+		os.Exit(1)
+	}
+	labels := strings.Split(string(buf[:]), "\n")
+	sort.Slice(labels, func(i, j int) bool { return labels[i] < labels[j] })
+
+	// Write the output to stdout
+	fmt.Println(`# Code generated by generate-test-suites, DO NOT EDIT.
+# gazelle:proto_strip_import_prefix /pkg
+
+ALL_TESTS = [`)
+	for _, label := range labels {
+		if len(label) > 0 {
```

Comment:
Curious why we needed this filter.

Reply:
Ah, because otherwise an empty label ends up in this list -- splitting on newlines results in an empty label ending up in the list, I think because the output of the command ends in `\n`.